backend/capellacollab/core/database/migration.py

```diff
@@ -75,8 +75,7 @@ def create_tools(db):
     registry = config["docker"]["registry"]
     capella = Tool(
         name="Capella",
-        docker_image_template=f"{registry}/t4c/client/remote:$version-latest",
-        docker_image_backup_template=f"{registry}/t4c/client/backup:$version-latest",
+        docker_image_template=f"{registry}/capella/remote:$version-latest",
```

Comment:
Any reason for this change? We use the T4C images for the persistent workspaces in both instances. And the deletion of the `docker_image_backup_template` will break the backups integration.

Reply:
I misread that as the readonly image.

The capella images are built by default. For the T4C images the (new) user has to do more work. I think it makes sense to do the default setup as new-user friendly as possible.

Reply:
On the other hand, it has to changed manually for each production version. The best case would be to detect if it was a Capella deployment with or without T4C. 

I'd rather argue that 90% of the users want to use the Collab Manager with TeamForCapella, so 90% would have to change it. For the other 10% of users, we can add a sentence to the README that it has to be changed.